install.sh

```diff
@@ -10,39 +11,90 @@ elif [ $PLATFORM == "Linux" ]; then
 	PLATFORM="ubuntu-20_04-x86-64"
 else
 	echo "Scripted installation on your platform is not supported."
-	echo "See compiled binaries in the latest release: https://github.com/simplex-chat/simplex-chat/releases/latest"
+	echo "See compiled binaries in the latest release: https://github.com/$APP_NAME/$APP_NAME/releases/latest"
 	exit 1
 fi
 
-[ ! -d $TARGET_DIR ] && mkdir -p $TARGET_DIR
+# / Prepare to upgrade from v0 to v1
+
+# Determine path of chat binary
+if [[ -n "$(which $APP_NAME)" ]]; then
+	binary=$(which $APP_NAME)
+elif [[ -f "$BIN_PATH" ]]; then
+	binary=$BIN_PATH
+else
+	binary=""
+fi
+
+# If chat binary not found, check v0 initial migration and offer to abort or continue
+if [[ -z $binary ]]; then
+	agent_db="$HOME/.simplex/simplex.agent.db"
+	if [[ \
+		-f "$agent_db" && \
+		$(echo "select * from migrations;" | sqlite3 $agent_db | grep 20210101_initial) \
+	]]; then
+		echo "Found initial migration of the previous version of SimpleX Chat, current version v1.0.0 is not compatible."
+		echo "If you choose to continue the installation current version will be installed as $APP_NAME with clean database, old database will be preserved."
+		echo "The next version v1.1.0 will be backwards compatible with your groups and contacts. Please see <link> for more information."
+		while true; do
+			read -p "Please choose to (a)bort or (c)ontinue: " yn
+			case $yn in
+					[Aa]* ) exit 1 ;;
+					[Cc]* ) break ;;
+					* ) echo "Please answer a or c."
+			esac
+		done
+	fi
+# If chat binary found, check version and offer to abort or continue, on continue rename chat binary
+elif [[ ! $($binary -h | grep v1) ]]; then
+	echo "Found previous version of SimpleX Chat, current version v1.0.0 is not compatible."
+	echo "If you choose to continue the installation it will be renamed to $APP_NAME-v0 and version v1 will be installed as $APP_NAME with clean database."
+	echo "The next version v1.1.0 will be backwards compatible with your groups and contacts. Please see <link> for more information."
```

Comment:
```suggestion
	echo "If you continue, it will be renamed to $APP_NAME-v0, and the new version will be installed as $APP_NAME with a clean database."
```